frontend/src/app/main/ui/dashboard/team.scss

```diff
@@ -308,8 +320,23 @@
   width: 100%;
 }
 
+.invitations-actions {
+  display: flex;
+  justify-content: end;
+  align-items: center;
+  gap: $s-16;
```

Comment:
Please Update this file following [SCSS migration guide](https://docs.google.com/document/d/1Z96Diqaxp0K5Nr-FoS3NN1LlGz_DIU-6eSENHgAiGug/edit?tab=t.0) using new spacig variables and typography mixin.